src/components/Gradebook/GradebookFilters/AssignmentGradeFilter/__snapshots__/test.jsx.snap

```diff
@@ -9,14 +9,14 @@ exports[`AssignmentGradeFilter Component snapshots buttons and groups disabled i
     id="assignmentGradeMin"
     label="Min Grade"
     onChange={[MockFunction handleSetMin]}
-    value="1"
+    value="2"
   />
   <PercentGroup
     disabled={true}
     id="assignmentGradeMax"
     label="Max Grade"
     onChange={[MockFunction handleSetMax]}
-    value="100"
+    value="98"
```

Comment:
These snapshot changes are not noteworthy.  Happening because we are more-accurately mapping to the test data.